tests/test_il_sex_offender_management.py

```diff
@@ -0,0 +1,94 @@
+from datetime import datetime
+from os.path import dirname, join
+
+from city_scrapers_core.constants import BOARD, CANCELLED
+from city_scrapers_core.utils import file_response
+from freezegun import freeze_time
+
+from city_scrapers.spiders.il_sex_offender_management import (
+    IlSexOffenderManagementSpider,
+)
+
+test_pdf_response = file_response(
+    join(dirname(__file__), "files", "il_sex_offender_management.pdf"),
+    url=(
+        "https://www2.illinois.gov/idoc/Documents/SOMB%20Meeting%20Agenda%202019%20August.pdf"  # noqa
+    ),
+    mode="rb",
+)
+
+test_response = file_response(
+    join(dirname(__file__), "files", "il_sex_offender_management.html"),
+    url="https://www2.illinois.gov/idoc/Pages/SexOffenderManagementBoard.aspx",
+)
+spider = IlSexOffenderManagementSpider()
+
+freezer = freeze_time("2020-12-12")
+freezer.start()
+
+parsed_items = [item for item in spider.parse(test_response)]
```

Comment:
To keep tests consistent we want to make sure that they're only parsing fixture files and not making any network requests. Usually the way we'll do this is to split out parsing methods from methods that `yield` requests so that we can parse fixture responses without having to chain requests together. One way to test this is to try and run your test file with wifi turned off on your computer and see if it still runs.

You can see an example of this setup in the tests for  [`chi_human_relations`](https://github.com/City-Bureau/city-scrapers/blob/main/tests/test_chi_human_relations.py)

Reply:
Hello, I actually was referencing `chi_human_relations` when I first wrote the test file! Like `chi_human_relations` test file, I downloaded the pdf in addition to the html that was already downloaded and ran the response on those files. Running pytest without wifi seems to work as it should with the current setup as well.  Could you provide some more details on where the problem area may be?

Reply:
Gotcha, thanks for clarifying! Ordinarily we're looking to check the items from the first run of `parse`, but you're right that since you're only checking the count and then parsing the PDF separately that this works without it. I'll resolve this